lang/java/reef-runtime-yarn/src/test/java/org/apache/reef/runtime/yarn/driver/YarnResourceRequestHandlerTest.java

```diff
@@ -57,70 +61,75 @@ public void onContainerRequest(AMRMClient.ContainerRequest... containerRequests)
+        requestOne.getMegaBytes(),
+        containerRequestHandler.getRequests()[0].getCapability().getMemory());
+
+    evaluatorRequestor.submit(requestTwo);
+    Assert.assertEquals("Request in REEF and YARN form should have the same amount of memory",
+        requestTwo.getMegaBytes(),
+        containerRequestHandler.getRequests()[0].getCapability().getMemory());
+
+    evaluatorRequestor.submit(requestOne);
+    Assert.assertNotEquals("Request in REEF and YARN form should have the same amount of memory",
+        requestTwo.getMegaBytes(),
+        containerRequestHandler.getRequests()[0].getCapability().getMemory());
   }
 
   @Test
   public void testEvaluatorCount() throws InjectionException {
-    final LoggingScopeFactory loggingScopeFactory = Tang.Factory.getTang().newInjector().getInstance(LoggingScopeFactory.class);
-    final EvaluatorRequestor evaluatorRequestor = new EvaluatorRequestorImpl(resourceCatalog, resourceRequestHandler, loggingScopeFactory);
+
+    final LoggingScopeFactory loggingScopeFactory =
+        Tang.Factory.getTang().newInjector().getInstance(LoggingScopeFactory.class);
+
+    final EvaluatorRequestor evaluatorRequestor =
+        new EvaluatorRequestorImpl(resourceCatalog, resourceRequestHandler, loggingScopeFactory);
+
     final EvaluatorRequest requestOne = EvaluatorRequest.newBuilder()
         .setNumber(1)
         .setMemory(64)
         .setNumberOfCores(1)
         .build();
+
     final EvaluatorRequest requestTwo = EvaluatorRequest.newBuilder()
         .setNumber(2)
         .setMemory(128)
         .setNumberOfCores(2)
         .build();
 
-      evaluatorRequestor.submit(requestOne);
-      Assert.assertEquals("Request in REEF and YARN form should have the same number of Evaluators",
-          requestOne.getNumber(),
-          containerRequestHandler.getRequests().length
-      );
-
-      evaluatorRequestor.submit(requestTwo);
-      Assert.assertEquals("Request in REEF and YARN form should have the same number of Evaluators",
-          requestTwo.getNumber(),
-          containerRequestHandler.getRequests().length
-      );
-
-      evaluatorRequestor.submit(requestTwo);
-      Assert.assertNotEquals("Request in REEF and YARN form should have the same number of Evaluators",
-          requestOne.getNumber(),
-          containerRequestHandler.getRequests().length
-      );
+    evaluatorRequestor.submit(requestOne);
+    Assert.assertEquals("Request in REEF and YARN form should have the same number of Evaluators",
+        requestOne.getNumber(),
+        containerRequestHandler.getRequests().length);
+
+    evaluatorRequestor.submit(requestTwo);
+    Assert.assertEquals("Request in REEF and YARN form should have the same number of Evaluators",
+        requestTwo.getNumber(),
+        containerRequestHandler.getRequests().length);
+
+    evaluatorRequestor.submit(requestTwo);
+    Assert.assertNotEquals("Request in REEF and YARN form should have the same number of Evaluators",
```

Comment:
Can you please update the message in assertNotEquals to reflect what it actually checks?